app_server/app/services/StreamService.js

```diff
@@ -72,6 +78,14 @@ Class.getListOfStreams = function(filters) {
   });
 };
 
+Class.stopStream = function(appName, appInstance, streamId) {
+  // TODO: Update stream's state in database before calling the below
```

Comment:
So the client will call `stopStream` and in this method we will call another method toggle the `live` status. Am I right to say that?